pkg/controllers/managementuser/rbac/handler_base.go

```diff
@@ -428,7 +428,7 @@ type createFn func(objectMeta metav1.ObjectMeta, subjects []rbacv1.Subject, role
 type listFn func(ns string, selector labels.Selector) ([]interface{}, error)
 type convertFn func(i interface{}) (string, string, []rbacv1.Subject)
 
-func (m *manager) ensureBindings(ns string, roles map[string]*v3.RoleTemplate, binding interface{}, client *objectclient.ObjectClient,
+func (m *manager) ensureBindings(ns string, roles map[string]*v3.RoleTemplate, binding metav1.Object, client *objectclient.ObjectClient,
```

Comment:
Is this necessary? Otherwise it may be out of scope.

Reply:
See the second commit. There is an issue that you run into when running Rancher locally in Goland and giving owner access to a cluster to another user.

Without this change, the only way to stop Goland from pausing constantly is to recompile.